### PR TITLE
Create Paradox-QuietLight.tmLanguage.json

### PR DESCRIPTION
**Pull request by UristMcDorf**
Yeah this is not finished at all but better than just looking at a field of brown because Quiet Light specifically has problems with the tokenisation of the language, apparently.

### CODE REVIEW

File: themes/Paradox-QuietLight.tmLanguage.json [hunk 1]
```diff
@@ -0,0 +1,391 @@
+// This is absolutely a work in progress. I can't properly see how everything is supposed to be tokenised until all effects/triggers/everything is defined properly.
+// Plus maybe some adjustments to the basic tokenisation are in order - but that's, again, only after everything is done at least for the first pass.
+// For now it's just a light theme that doesn't necessarily look like crap, just a bit of a colour mismatch everywhere. So that's a good thing, I hope.
+{
+	"name": "CKII - Quiet Light",
```
Comment (tboby) [inline themes/Paradox-QuietLight.tmLanguage.json:5]: Make the name Paradox, it'll work in all the games :)

File: themes/Paradox-QuietLight.tmLanguage.json [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+		{
+			"scope": "token.debug-token",
+			"settings": {
+				"foreground": "#800080"
+			}
+		},
+		{
+			"scope": "variable.language.multi_scopes.paradox",
+			"settings": {
+				"foreground": "#4B69C6" // Keywords? Some fields in classes use this, some use variable.language.definition_tokens.paradox; weird
```
Comment (tboby) [inline themes/Paradox-QuietLight.tmLanguage.json:285]: Are inline comments valid in json?

File: themes/Paradox-QuietLight.tmLanguage.json (hunk 1 above)
Comment (UristMcDorf) [reply themes/Paradox-QuietLight.tmLanguage.json:5]: I just followed the Stellaris' theme convention :)

File: themes/Paradox-QuietLight.tmLanguage.json (hunk 2 above)
Comment (UristMcDorf) [reply themes/Paradox-QuietLight.tmLanguage.json:285]: Don't know about json proper but everything worked fine in VS Code when I edited the settings and placed inline comments next to them.

**Pull request merged**